payara/Payara
# Fish 7217 remote ejb jndi lookup fails on jdk 11 and 17

## Description
Remote EJB JNDI Lookup fails on JDK 17

## Important Info
### Blockers
None

## Testing
### New tests
None

### Testing Performed
1. Download -Payara-6.2023.2-JNDI-Issue-master.zip from https://payara.atlassian.net/browse/FISH-7217
2. unzip -Payara-6.2023.2-JNDI-Issue-master.zip
3. cd Paya-Payara-6.2023.2-JNDI-Issue-master
4. set JAVA_HOME to JDK 17 (C:\Program Files\Zulu\zulu-17) 
5. create GLASSFISH_HOME and point to Payara6 folder (C:\Users\luise\git\Payara\appserver\distributions\payara\target\stage\payara6)
6. ant setup
7. ant compile
8. ant deploy
9. Poke http://localhost:8080/app-web/myservlet
10. Should return Servlet: Hello Servlet World!
Payara Server 6.2023.7 #badassfish com.test.ejb._TestSessionBeanRemote_Wrapper@56c3d99a

### Testing Environment
Zulu JDK 17 on Windows 11 with Maven 3.6.0

## Documentation
None

## Notes for Reviewers
None
